speculative_decoding/config.yaml

```diff
@@ -1,18 +1,21 @@
 base_image:
-  image: nvcr.io/nvidia/tritonserver:24.01-trtllm-python-py3
-  python_executable_path: /usr/bin/python3
+  image: "nvcr.io/nvidia/tritonserver:24.01-trtllm-python-py3"
```

Comment:
nit: Do you need quotes for this yaml spec? I think as long as the values are not to be confused with some part of the yaml syntax, it's not needed.